src/Context/LoginContext.php

```diff
@@ -315,14 +316,21 @@ protected function generateMember($email, $password, $group, $identifier)
 
         // make sure any validation for password is skipped, since we're not testing complexity here
         $validator = Member::password_validator();
-        Member::set_password_validator(null);
+        $nullValidator = new PasswordValidator();
+        $nullValidator->setMinLength(0);
+        $nullValidator->setMinTestScore(0);
+        $nullValidator->setHistoricCount(0);
+        Member::set_password_validator($nullValidator);
```

Comment:
Without this change, we get this error:
> You've already used that password in the past, please choose a new password

The comment makes it clear this was the intention here.
Outside of testing I can't see any reason to _not_ have a member password validator, and the current behaviour is when `null` is passed in the result is the default password validator is instantiated which IMO is good behaviour.